src/main/resources/templates/AdvisorOpenshift/insightsEmailBody.html

```diff
@@ -580,7 +580,7 @@ <h1><b>{#insert content-title}{/}</b></h1>
                                 <tbody>
                                 <tr>
                                     <td class="rh-footer__text">
-                                        This email was sent by Red Hat Insights |<a href="https://cloud.redhat.com/user-preferences/email" target="_blank">&nbsp;Manage notification preferences</a>
+                                        This email was sent by Red Hat Insights |<a href="https://cloud.redhat.com/user-preferences/email" target="_blank">&nbsp;Manage email preferences</a>
```

Comment:
Thanks for reminding me that one.
You are right, it should. But right now in production I don't see the "Openshift > Notifications" tabs.
I'll update the templates to include a link to https://cloud.redhat.com/user-preferences/notifications/openshift. But it might not work for some time =)